doc/ug/getting_started_sw.md

```diff
@@ -42,4 +42,4 @@ Building an executable `foo` destined to run on an OpenTitan device (i.e., under
 * `foo.vmem`: a Verilog memory file which can be read by `$readmemh()` in Verilog code.
 
 Building an executable destined to run on a host machine (i.e., under `sw/host`) will output a host excecutable under `build-bin/sw/host`, which can be run directly.
-Currently, each "platform" (`fpga`, `sim-verolator`, etc) have their own copies of all host targets; this is a limitation of our Meson setup, and they are otherwise indistinguishable.
\ No newline at end of file
+Currently, each "platform" (`fpga`, `sim-verolator`, etc) have their own copies of all host targets; this is a limitation of our Meson setup, and they are otherwise indistinguishable.
```

Comment:
Since you are changing this line anyway, could you please also fix this typo sim-verolator -> sim-verilator ?

Reply:
fixed. will merge if still passes checks